cirq-core/cirq/contrib/graph_device/graph_device.py

```diff
@@ -155,6 +155,7 @@ def __init__(
     def qubits(self) -> Tuple['cirq.Qid', ...]:
         return cast(Tuple['cirq.Qid', ...], tuple(sorted(self.device_graph.vertices)))
 
+    @_compat.deprecated(fix='Check metadata.qubit_set if applicable.', deadline='v0.15')
```

Comment:
Right, I opted to not implement the metadata for this class since it looks like it might be a non-trivial amount of work (hypergraphs vs regular graphs), so opting to recommend we use the qubits property and leave this as is. Updated the wording in the deprecation.